examples/pygfx-example.py

```diff
@@ -168,34 +119,16 @@ def toggle():
             # When increasing this number, it will take longer
             # and longer for pygfx to create the render pipeline
             # (compiling shaders and such), so be careful...
-            # {"count": 20},
+            {"count": 50},
         ),
         h(
-            Landmark,
-            {
-                "name": "funk",
-            },
-        ),
-        h(
-            Landmark,
-            {
-                "position": [4, 4, -4],
-            },
-        ),
-        h(
-            "Point",
+            "Mesh",
             {
                 "name": "Hip",
                 "position": [2, 2, 2],
                 "color": [1, 0.4, 0],
```

Comment:
color on Mesh directly? what does that do?

Reply:
I think this is still some leftover code :) I'll remove it!